scripts/CIP-24.ts

```diff
@@ -0,0 +1,70 @@
+// run with
+// npx hardhat run ./scripts/CIP-24.ts --network hardhat
+import hre, { deployments, network, hardhatArguments } from "hardhat";
+import {
+  castVote,
+  createProposal,
+  executeProposal,
+  fundMultisign,
+  queueProposal,
+} from "./utils";
+import { BigNumber } from "ethers";
+
+async function main() {
+  const ethers = hre.ethers;
+  let multisig = "0xa70b638B70154EdfCbb8DbbBd04900F328F32c35";
+  let ctxAmount = ethers.utils.parseEther("215750");
+  let ctx = await deployments.get("Ctx");
+  let ctxContract = await ethers.getContractAt("Ctx", ctx.address);
+
+  const abi = new ethers.utils.AbiCoder();
+  const targets = [ctx.address];
+  const values = [BigNumber.from(0)];
+  const signatures = ["transfer(address,uint256)"];
+  const calldatas = [
+    abi.encode(["address", "uint256"], [multisig, ctxAmount]),
+  ];
+  const description =
+    "CIP-24: Operating Expenditures 2024";
```

Comment:
Should this be: "Operating Expenditures Q1 2024"?
Not sure if this is for the entire year or just Q1 2024.